packages/core/src/components/link/link.stories.js

```diff
@@ -0,0 +1,92 @@
+import './link.css';
+import '../../icons/icon.css';
+
+export default {
+  title: 'Components/Link',
+  decorators: [(storyFn) => `<div style="max-width: 400px">${storyFn()}</div>`],
+};
+
+export const Default = () => `
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--small">Default link</a>
+`;
+
+
+export const InternalLinks = () => `
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--small">Internal link size small</a>
+  <br/>
+  <br/>
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--medium">Internal link size medium</a>
+  <br/>
+  <br/>
+  <a href="/?path=/story/components-link--all" class="hds-link hds-link--large">Internal link size large</a>
+`;
+
+InternalLinks.storyName = 'Internal links';
+
+export const ExternalLinks = () => `
+  <a href="https://hds.hel.fi" class="hds-link hds-link--small" aria-label="External link size small. Opens in external domain">
+    External link size small<i class="hds-icon icon hds-icon--link-external hds-icon--size-xs vertical-align-small-or-medium-icon" aria-hidden="true"></i></a>
+  <br/>
+  <br/>
+  <a href="https://hds.hel.fi" class="hds-link hds-link--medium" aria-label="External link size medium. Opens in external domain">
+    External link size medium<i class="hds-icon icon hds-icon--link-external hds-icon--size-s vertical-align-small-or-medium-icon" aria-hidden="true"></i></a>
+  <br/>
+  <br/>
+  <a href="https://hds.hel.fi" class="hds-link hds-link--large" aria-label="External link size large. Opens in external domain">
+    External link size large<i class="hds-icon icon hds-icon--link-external hds-icon--size-l vertical-align-big-icon" aria-hidden="true"></i>
+  </a>
+`;
+
+ExternalLinks.storyName = 'External links';
+
+export const openInNewTabLink = () => `
+  <a href="https://hds.hel.fi" class="hds-link hds-link--small" target="_blank" aria-label="Link that opens in a new tab. Opens in a new tab. Opens in external domain">
```

Comment:
I guess it would still be good to add "rel=noopener" with "target=_blank". It is fixed in some browsers but not in all.
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Reply:
- [ ] Add rel=noopener